executor/executor_test.go

```diff
@@ -8288,3 +8288,15 @@ func (s *testSuite1) TestIssue24091(c *C) {
 	tk.MustQuery("select null div 0;").Check(testkit.Rows("<nil>"))
 	tk.MustQuery("select * from t;").Check(testkit.Rows("<nil>"))
 }
+
+func (s *testSuite1) TestIssue24210(c *C) {
```

Comment:
Please use `SerialTestSuite` if you use failpoints.

Reply:
addressed